packages/enhanced/src/lib/container/ContainerEntryModule.ts

```diff
@@ -170,6 +176,12 @@ class ContainerEntryModule extends Module {
       ) as unknown as Dependency,
     );
 
+    this.addDependency(
+			new EntryDependency(
+				FederationRuntimePlugin.getFilePath(this._runtimePlugins)
```

Comment:
This is brilliant. Well done. Such a clean way to inject middlewares into the runtime